src/main.rs

```diff
@@ -565,6 +633,7 @@ async fn run() -> Result<()> {
                       version = {version:?};
 
                       src = {src_expr};
+                      {source_root_expr}
```

Comment:
This will create an extra line of spaces without `sourceRoot`, the newline and spaces should be part of `source_root_expr` instead

```suggestion
                      src = {src_expr};{source_root_expr}
```